MOBILE_API.xml

```diff
@@ -647,44 +647,106 @@
+        </element>
+        <element name="FAN_DOWN" since="4.5">
+            <description>Tied to CLIMATE modules.</description>
+        </element>
+        <element name="TEMP_UP" since="4.5">
+            <description>Tied to CLIMATE modules.</description>
+        </element>
+        <element name="TEMP_DOWN" since="4.5">
+            <description>Tied to CLIMATE modules.</description>
+        </element>
+        <element name="DEFROST_MAX" since="4.5">
+            <description>Tied to CLIMATE modules.</description>
+        </element>
+        <element name="DEFROST" since="4.5">
+            <description>Tied to CLIMATE modules.</description>
+        </element>
+        <element name="DEFROST_REAR" since="4.5">
+            <description>Tied to CLIMATE modules.</description>
+        </element>
+        <element name="UPPER_VENT" since="4.5">
+            <description>Tied to CLIMATE modules.</description>
+        </element>
+        <element name="LOWER_VENT" since="4.5">
+            <description>Tied to CLIMATE modules.</description>
+        </element>
         <!-- Radio Buttons -->
-        <element name="VOLUME_UP" since="4.5" />
-        <element name="VOLUME_DOWN" since="4.5" />
-        <element name="EJECT" since="4.5" />
-        <element name="SOURCE" since="4.5" />
-        <element name="SHUFFLE" since="4.5" />
-        <element name="REPEAT" since="4.5" />
+        <element name="VOLUME_UP" since="4.5">
+            <description>Tied to RADIO modules.</description>
+        </element>
+        <element name="VOLUME_DOWN" since="4.5">
+            <description>Tied to RADIO modules.</description>
+        </element>
+        <element name="EJECT" since="4.5">
+            <description>Tied to RADIO modules.</description>
+        </element>
+        <element name="SOURCE" since="4.5">
+            <description>Tied to RADIO modules.</description>
+        </element>
+        <element name="SHUFFLE" since="4.5">
+            <description>Tied to RADIO modules.</description>
+        </element>
+        <element name="REPEAT" since="4.5">
+            <description>Tied to RADIO modules.</description>
+        </element>
         <!-- Navigation Buttons -->
-        <element name="NAV_CENTER_LOCATION" since="6.0" />
-        <element name="NAV_ZOOM_IN" since="6.0" />
-        <element name="NAV_ZOOM_OUT" since="6.0" />
-        <element name="NAV_PAN_UP" since="6.0" />
-        <element name="NAV_PAN_UP_RIGHT" since="6.0" />
-        <element name="NAV_PAN_RIGHT" since="6.0" />
-        <element name="NAV_PAN_DOWN_RIGHT" since="6.0" />
-        <element name="NAV_PAN_DOWN" since="6.0" />
-        <element name="NAV_PAN_DOWN_LEFT" since="6.0" />
-        <element name="NAV_PAN_LEFT" since="6.0" />
-        <element name="NAV_PAN_UP_LEFT" since="6.0" />
+        <element name="NAV_CENTER_LOCATION" since="6.0">
+            <description>Tied to NAVIGATION modules.</description>
```

Comment:
There is no `NAVIGATION` remote control module

Reply:
Good catch. Fixed in https://github.com/smartdevicelink/rpc_spec/pull/290/commits/bdbb236b620e46efb451cf4204aa0c955f9598b6